site/docs/latest/functions/quickstart.md

```diff
@@ -212,6 +212,14 @@ If you see `Deleted successfully` in the output, then you've succesfully run, up
 
 ## Writing and running a new function
 
+{% include admonition.html type="info" content="
+In order to write and run the [Python](../api#python) function below, you'll need to install the [`grpc`](https://pypi.org/project/grpc/) and [`protobuf`](https://pypi.org/project/protobuf/) libraries. To install them using [pip]():
+
+```bash
+$ pip install grpc protobuf
```

Comment:
the actual list is 
pip install pulsar-client protobuf futures grpcio grpcio-tools